test/ntttcp_test.py

```diff
@@ -0,0 +1,441 @@
+import re
+import time
+import json
+import pytest
+import subprocess
+from typing import Optional, Tuple
+from xml.sax import make_parser
+from xml.sax.handler import ContentHandler
+
+class NtttcpResult:
+    def __init__(
+        self,
+        rec_out : str,
+        send_out : str,
+        ) -> None:
+        self.rec_stdout = rec_out
+        self.send_stdout = send_out
+
+class ParseNtttcpResult:
+    throughput_pattern = re.compile(
+        r"INFO: \\t throughput\\t:(\d+.\d+)Gbps"
+    )  # need to refine, for the throughput is not always Gbps
+    daemon_pattern = re.compile(r"\*\*\*\s+run as daemon")
+    multi_clients_pattern = re.compile(r"multiple clients:\s+yes")
+    epoll_pattern = re.compile(r"\*\*\*\s+use epoll()")
+    system_tcp_retransmit_pattern = re.compile(r"show system tcp retransmit:\s+yes")
```

Comment:
We should also judge 
03:08:43 INFO: tcp retransmit:
03:08:43 INFO:   retrans_segments/sec   :309.53

Reply:
done